lua/wire/wireshared.lua

```diff
@@ -777,6 +777,23 @@ function WireLib.levenshtein( s, t )
 	return d[#d]
 end
 
+--- Escape a string so it can be searched for literally with Lua's pattern functions.
+function WireLib.PatternEscape(literal)
```

Comment:
this function actually already exists in the string library http://wiki.garrysmod.com/page/string/PatternSafe